app/models/user.rb

```diff
@@ -102,13 +102,9 @@ def self.from_omniauth(access_token)
   # (e.g. assigned to current line and in user.patients)
 
   def recently_called_patients(lines = LINES)
-    patients.in(_line: lines)
-            .select { |patient| recently_called_by_user? patient }
-  end
-
-  def call_list_patients(lines = LINES)
-    patients.in(_line: lines)
-            .reject { |patient| recently_called_by_user? patient }
+    Patient.in(_line: lines, user_ids: BSON::ObjectId(id)).where( :$and =>
```

Comment:
Here's what I gathered based on the initial DB call.

original call
```
MONGODB | localhost:27017 | dcaf_case_management_development.find | STARTED | {"find"=>"patients", "filter"=>{"$and"=>[{"_id"=>{"$in"=>[BSON::ObjectId('5950a1542cde110a61906299'), BSON::ObjectId('5950a1542cde110a6190629d'), BSON::ObjectId('5950a1542cde110a619062a1'), BSON::ObjectId('5950a1542cde110a619062a5'), BSON::ObjectId('5...
MONGODB | localhost:27017 | dcaf_case_management_development.find | SUCCEEDED | 0.010525s
```

With the new DB calls
```
MONGODB | localhost:27017 | dcaf_case_management_development.find | STARTED | {"find"=>"patients", "filter"=>{"_line"=>{"$in"=>[:DC]}, "user_ids"=>{"$in"=>[BSON::ObjectId('5950a1532cde110a6190628b')]}, "$and"=>[{"calls.updated_at"=>{"$ne"=>{"$gte"=>2017-06-25 22:03:00 UTC, "$lte"=>2017-06-26 06:03:00 UTC}}}, {"calls.created_by_...
MONGODB | localhost:27017 | dcaf_case_management_development.find | SUCCEEDED | 0.007338000000000001s
```

There looks to be a difference of 1 significant figure, so that's 👍